src/main/java/com/nike/cerberus/command/core/CreateCompleteCerberusDataS3BackupCommand.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.command.core;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.nike.cerberus.command.Command;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.operation.core.CreateCompleteCerberusDataS3BackupOperation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.nike.cerberus.command.core.CreateCompleteCerberusDataS3BackupCommand.COMMAND_NAME;
+
+
+/**
+ * Command for creating Safe Deposit Box Metadata and Vault secret data backups in S3 encrypted with KMS
+ */
+@Parameters(
+        commandNames = COMMAND_NAME,
+        commandDescription = "Allows Cerberus operators to create a complete backup in S3 encrypted with KMS."
+)
+public class CreateCompleteCerberusDataS3BackupCommand implements Command {
```

Comment:
nitpick: can we go with a shorter class name?  CreateBackupCommand is what I would look for